runelite-client/src/main/java/net/runelite/client/plugins/npcunaggroarea/NpcAggroAreaOverlay.java

```diff
@@ -66,7 +66,7 @@ private NpcAggroAreaOverlay(Client client, NpcAggroAreaConfig config, NpcAggroAr
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
-		if (!plugin.isActive() || plugin.getSafeCenters()[1] == null)
+		if (!plugin.isActive() || plugin.getSafeCenters()[0] == null || plugin.getSafeCenters()[1] == null)
```

Comment:
I don't think this line should be changed. It is helpful to know when and where the first center is placed even before the second center is calibrated. The warning overlay can (and should) still be shown until the second center is set though.

Reply:
Thing is, with how it currently works, it won't always show where the first centre is; it only happen if for some reason, the second is calibrated first. I think making it consistently wait until both are set is better than sometimes showing information to the user which is wrong (since a half-calibration will not save)